docs/articles/nunit/writing-tests/attributes/cancelafter.md

```diff
@@ -0,0 +1,52 @@
+# CancelAfter
+
+Normally, NUnit simply runs tests and waits for them to terminate—the test is allowed to run indefinitely. For certain kinds of tests, however, it may be desired to specify a timeout value.
+
+For .NET Core and later, [`Thread.Abort`](https://learn.microsoft.com/en-us/dotnet/api/system.threading.thread.abort?view=net-7.0) as used by the [`TimeoutAttribue`](./timeout.md) can no longer be used and there is no way to interrupt a endless loop.
+
+For all tests, one could use the `--blame-hang(-timeout)`  options of [`dotnet test`](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-test#options)
+however this will stop any further exection of the remaining tests.
+
+To still be able to cancel tests, one has to move to cooperative cancellation. See [Cancellation in Managed Threads](https://learn.microsoft.com/en-us/dotnet/standard/threading/cancellation-in-managed-threads) using a [`CancellationToken``](https://learn.microsoft.com/en-us/dotnet/api/system.threading.cancellationtoken?view=net-7.0).
+
+The **CancelAfterAttribute** is used to specify a timeout value in milliseconds for a test case.
```

Comment:
```suggestion
The `CancelAfterAttribute` is used to specify a timeout value in milliseconds for a test case.
```

Reply:
Again, this was the emphasis used in timout.md and also in various other attributes descriptions. 
If we change this, we should align the others, or keep this as it stands.

Reply:
Aligned.

Reply:
I've been suggesting changes whenever they come up because I haven't had time to do a whole pass of the docs yet but I welcome a PR for that too 👍